real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/logging/OldLogConverter.java

```diff
@@ -0,0 +1,183 @@
+package com.roboclub.robobuggy.logging;
+import java.util.*;
+import java.io.*;
+
+
+public class OldLogConverter {
+	
+/** Steering Angle Conversion Rate */
+static private final int ARD_TO_DEG = 100;
+/** Steering Angle offset?? */
+static private final int OFFSET = -200;
+	
+static private ArrayList<String> sensor_type = new ArrayList<String>();
+static private ArrayList<Integer> sensor_quant = new ArrayList<Integer>();
+
+	public static void main(String[] args) throws IOException {
+		boolean fileinput = false;
+		Scanner scanner; PrintStream writer;
+		
+		if(fileinput){
```

Comment:
So that it becomes a utility, rather than a standalone program that can only run once